drivers/SmartThings/matter-switch/src/test_matter_button_switch_mcd.lua

```diff
@@ -0,0 +1,199 @@
+-- Copyright 2024 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local test = require "integration_test"
+local t_utils = require "integration_test.utils"
+local capabilities = require "st.capabilities"
+local utils = require "st.utils"
+local dkjson = require "dkjson"
+
+local clusters = require "st.matter.clusters"
+
+local button1_ep = 10
+local button2_ep = 20
+local light_ep = 30
+
+local mock_device = test.mock_device.build_test_matter_device({
+  label = "Matter Switch",
+  profile = t_utils.get_profile_definition("2-button-battery-switch.yml"),
```

Comment:
Is this a real profile? I don't see it here or on main

Reply:
This test file and that profile are leftovers from experimenting with other types of configurations, and so this file should be removed.